tox.ini

```
@@ -42,6 +45,7 @@ deps =
 commands =
     python --version
     dj{32,40,41,42,50}: pytest --no-migrations -o DJANGO_SETTINGS_MODULE=tests.django.settings -o django_find_project=false {posargs:tests/core/ tests/django}
+    fa{100}: pytest tests/core/ tests/fastapi/ {posargs:}
```

Based on the other test commands, should this be:

```suggestion
    fa{100}: pytest {posargs: tests/core/ tests/fastapi/}
```

?